src/Service/Logger.service.js

```diff
@@ -201,6 +207,23 @@ export default class LoggerService extends DependencyAwareClass {
     this.logger.log('info', this.processMessage(message));
   }
 
+  /**
+   * Logs an error, using logger.error
+   * or logger.info based on the logging levels
+   * that are based on the process.env.DEPLOY_ENV
+   *
+   * Please note that LoggerService.error and LoggerService.info
+   * have different signatures. The function uses the shared argument
+   * instead of introducing ambiguity.
+   *
+   * @param error
+   */
+  warning(error) {
+    const loggerFunction = LOGGING_LEVELS[process.env.DEPLOY_ENV] || 'error';
```

Comment:
Is `process.env.DEPLOY_ENV` better than `process.env.STAGE` for this sort of thing?

(Not sure if `DEPLOY_ENV` gets set when running `sls offline` but I _think_ STAGE does though not 100% sure)

Reply:
e421f27e8d45b1dc2d6c3ad5925bd281edd51d7d